sdcm/sct_events/base.py

```diff
@@ -210,6 +220,57 @@ def __del__(self):
                 print(warning)
 
 
+class InformationalEvent(SctEvent, abstract=True):
+
+    def __init__(self, severity: Severity = Severity.UNKNOWN):
+        super(InformationalEvent, self).__init__(severity=severity)
+        self.time_interval = EventInterval.Informational.value
+
+
+class ContinuousEvent(SctEvent, abstract=True):
+
+    def __init__(self,
+                 severity: Severity = Severity.UNKNOWN):
+        super().__init__(severity=severity)
+        self.log_file_name = None
+        self.errors = None
```

Comment:
make it list by default

Reply:
This value is overrided in the StressEvent class to None. Instead of change it to the "[]" in the StressEvent also I handle it's default value in the `sdcm.sct_events.base.ContinuousEvent.__exit__`